lib/raygun.ts

```diff
@@ -44,20 +45,7 @@ type SendOptionsResult =
 
 const debug = require("debug")("raygun");
 
-let apmBridge:
-  | undefined
-  | null
-  | { notify(e: string | Error, s: string): void } = undefined;
-
-try {
-  if (module.parent) {
-    apmBridge = module.parent.require("raygun-apm/lib/src/crash_reporting");
-  } else {
-    apmBridge = require("raygun-apm/lib/src/crash_reporting");
-  }
```

Comment:
The issue with the old code is that `module.parent` is deprecated: https://nodejs.org/api/deprecations.html#DEP0144

Then, the code attempts to load the module, but throws an error that appears as a build warning in webpack apps, e.g. `Module not found: Can't resolve 'raygun-apm/lib/src/crash_reporting' in ...`

That's why I think it is better to simply move this dependency outside the project and let users connect the two together, which requires an extra call.